gpytorch/lazy/batch_repeat_lazy_tensor.py

```diff
@@ -283,3 +286,22 @@ def repeat(self, *sizes):
                 for orig_repeat_size, new_repeat_size in zip(padded_batch_repeat, sizes[:-2])
             ),
         )
+
+    @cached(name="svd")
+    def _svd(self) -> Tuple["LazyTensor", Tensor, "LazyTensor"]:
+        U_, S_, V_ = self.base_lazy_tensor.svd()
+        U = U_.repeat(*self.batch_repeat, 1, 1)
+        S = S_.repeat(*self.batch_repeat, 1)
+        V = V_.repeat(*self.batch_repeat, 1, 1)
+        return U, S, V
+
+    @cached(name="symeig")
+    def _symeig(self, eigenvectors: bool = False) -> Tuple[Tensor, Optional[LazyTensor]]:
+        try:
+            return pop_from_cache(self, "symeig", eigenvectors=True)
```

Comment:
I guess *technically* for these "wrapper" LTs we could check the base LT for its cache too?

Reply:
Oh I see, calling `symeig` down below on the base class should handle that.